From ad207a6f83dac390b2d95e3a8262d27292921863 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Wed, 26 Jun 2019 11:49:33 +0800
Subject: [PATCH] dlm: fix compile error since xml2-config should not be used

xml2-config is disabled, so change Makefile to use pkgconfig
to find libxml2.

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 fence/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fence/Makefile b/fence/Makefile
index b927879..6e16078 100644
--- a/fence/Makefile
+++ b/fence/Makefile
@@ -18,11 +18,11 @@ CFLAGS += -D_GNU_SOURCE -O2 -ggdb \
 	-fstack-clash-protection -Wl,-z,now
 
 CFLAGS += -fPIE -DPIE
-CFLAGS += `xml2-config --cflags`
+CFLAGS += `pkg-config libxml-2.0 --cflags`
 CFLAGS += -I../include
 
 LDFLAGS += -Wl,-z,relro -pie
-LDFLAGS += `xml2-config --libs`
+LDFLAGS += `pkg-config libxml-2.0 --libs`
 LDFLAGS += -ldl
 
 all: $(BIN_TARGET)
-- 
2.7.4

